app/src/newdocs/components/CanvasModuleConfig/index.jsx

```diff
@@ -0,0 +1,135 @@
+/* eslint-disable flowtype/no-types-missing-file-annotation */
+import React from 'react'
+import isEmpty from 'lodash/isEmpty'
+import ReactMarkdown from 'react-markdown'
+
+import createMdSnippet from '$newdocs/utils/createMdSnippet'
+import styles from './canvasModuleConfig.pcss'
+
+const inputs = (moduleInputs, moduleInputsRaw) => {
+    const types = []
+    const defaultValues = []
+
+    if (moduleInputs && moduleInputsRaw) {
+        Object.entries(moduleInputs).map(([key]) =>
+            moduleInputsRaw.map((ins) => {
+                if (ins.name === key) {
+                    types.push(ins.type)
+                    defaultValues.push(ins.defaultValue)
+                }
+                return false
+            }))
+    }
+
+    return !isEmpty(moduleInputs)
+        ? createMdSnippet(moduleInputs, types, defaultValues)
+        : false
```

Comment:
Could `return !isEmpty(moduleInputs) && createMdSnippet(moduleInputs, types, defaultValues)` if you wanted a one-liner.